plugin/client_rpc.go

```diff
@@ -396,7 +397,7 @@ func (g *apiRPCClient) PluginHTTP(request *http.Request) *http.Response {
 
 	requestBody, err := ioutil.ReadAll(request.Body)
 	if err != nil {
-		log.Printf("RPC call to PluginHTTP API failed: %s", err.Error())
+		g.LogError("RPC call to PluginHTTP API failed: ", err.Error())
```

Comment:
`LogError` logs first a message and then a list of pairs. Hence this needs to be something like:
```suggestion
		g.LogError("RPC call to PluginHTTP API failed", "error", err.Error())
```

The same comment applies to some other places. Please double check the file.

Reply:
Done! Thanks @hanzei!